src/dialog/overflow-menu/overflow-menu.stories.ts

```diff
@@ -1,7 +1,18 @@
 import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs, boolean, number } from "@storybook/addon-knobs/angular";
 
 import { DialogModule } from "../../";
 
+let options;
+
+const createOptions = (count: number): Array<string> => {
```

Comment:
Should be a normal function if possible, to match other similar places in codebase.
```suggestion
const createOptions(count: number): Array<string> {
```